frontend/src/store/projectProvider.tsx

```diff
@@ -1,53 +1,80 @@
 // ProjectContext is not currently being used, but is kept here for future reference incase it is useful.
-
 import { useReducer } from "react";
+
 import ProjectContext from "./project-context";
+// import { Project } from "../shared/Interfaces";
+
+// const initialProjects: Project[] = [];
+
+enum ProjectActionKind {
+  StoreProjects = "STORE_PROJECTS",
+  AddProject = "ADD_PROJECT",
+}
+
+// interface ProjectState {
```

Comment:
nit: let's omit commented-out code unless there's a good reason to keep it around (like for future reference, in which case a comment would help describe that intention)

Reply:
Removed comments